com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/extensions/ForwardingResourceDescriptionStrategyMap.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Avaloq Evolution AG and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Avaloq Evolution AG - initial API and implementation
+ *******************************************************************************/
+
+package com.avaloq.tools.ddk.xtext.resource.extensions;
+
+import java.util.Map;
+
+import com.google.common.collect.ForwardingMap;
+import com.google.common.collect.ImmutableMap;
+
+
+public class ForwardingResourceDescriptionStrategyMap extends ForwardingMap<String, String> {
+
+  private Map<String, String> delegate;
+
+  protected void build(final ImmutableMap.Builder<String, String> builder) {

```

Comment:
perhaps rather something like 'addItems' or 'fill' or smth else? The actual map gets built later...

Reply:
done